src/Symfony/Component/Debug/DebugClassLoader.php

```diff
@@ -129,21 +129,19 @@ public static function disable()
     }
 
     /**
-     * @return string|null
+     * @param string $class
```

Comment:
this is wrong. You must not remove the the doc for the return type (and adding the doc for the param is useless now, as it duplicates the typehint)